### DISCUSSION TITLE
Authentication Expired for 127.0.0.1 error

### DISCUSSION THREAD

**Discussion by graymccord** [fuatakgun/eufy_security#1167] [General]
Started receiving this error yesterday, and of course all of the Eufy security devices are now not available in HA. The message tells me I need to reconfigure by entering a Captcha, which it presents to me and I believe I successfully enter. It then "works" for a minute or so before telling me its automatically updating in the background, but it never seems to complete whatever its updating because the integration never works.
Has anyone seen this? Have an idea for how to fix it?  Thanks!

**Comment by viko-repo** [comment]
![error](https://github.com/fuatakgun/eufy_security/assets/14986387/4ef232bd-8869-42a9-8d20-74c877b79c33)
As above, I have the same problem

**Reply by mhouser78** [reply]
This worked for me!

**Comment by Memaddog** [comment]
Same exact problem after updating to 8.0.4

**Comment by JelmerD** [comment]
Same issue on HAOS 2024.10

**Comment by thomasdg** [comment]
Had the same problem and per discussions in the issues pages fixed it:
- Changed password in Eufy Security app; my original password no longer was recognised as correct so requested a password reset link and entered the same old password as a new one.
- Entered the captcha again (maybe twice, don't remember) 
- All fixed!
